### ISSUE TITLE
Overwrite zip file members in doExtractMembers

### ISSUE THREAD

**Issue by hammat** [alchemy-fr/Zippy#96]
Hi !

I'm starting using Zippy to handle zip files, thanks for the job by the way.
If I try to extract a file that already exists, it simply triggers an exception.

Is there any reason extraction of members doesn't have the overwrite option (-o) while the the doExtract does have it ?

**Comment by aztech-dev** [comment]
Not sure actually. Changing that option would however be a BC break, so a sensible approach would be to preserve current behavior by default, and add an option in the library to allow overwrites on extraction.

**Comment by aztech-dev** [comment]
@hammat Can you provide sample code that reproduces your issue ?

**Comment by aztech-dev** [comment]
So I finally was able to reproduce your issue and pushed a fix on master (released as 0.3.3) that now allows you to do the following:

```php
$archive = $zippy->load('archive.zip');

foreach ($archive as $member) {
    // Passing true as a second argument will force overwriting of files
    $member->extract($destination, true);
}
```

Hope that helps

**Comment by hammat** [comment]
Oh my bad for my late answer, thanks again for your work !
